analyzer/testdata/src/checkers-default/empty/empty_test.go

```diff
@@ -9,109 +9,499 @@ import (
+		assert.Exactlyf(t, ``, b, "msg with args %d %s", 42, "42")                        // want "empty: use assert\\.Emptyf"
+		assert.Exactly(t, ``, string(b))                                                  // want "empty: use assert\\.Empty"
+		assert.Exactlyf(t, ``, string(b), "msg with args %d %s", 42, "42")                // want "empty: use assert\\.Emptyf"
+		assert.Exactly(t, ``, []string{"e"})                                              // want "empty: use assert\\.Empty"
+		assert.Exactlyf(t, ``, []string{"e"}, "msg with args %d %s", 42, "42")            // want "empty: use assert\\.Emptyf"
+		assert.Empty(t, len(elems))                                                       // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Empty\\)"
+		assert.Emptyf(t, len(elems), "msg with args %d %s", 42, "42")                     // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Emptyf\\)"
+		assert.Empty(t, len(str))                                                         // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Empty\\)"
+		assert.Emptyf(t, len(str), "msg with args %d %s", 42, "42")                       // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Emptyf\\)"
+		assert.Empty(t, len(string(str)))                                                 // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Empty\\)"
+		assert.Emptyf(t, len(string(str)), "msg with args %d %s", 42, "42")               // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Emptyf\\)"
+		assert.Empty(t, len(b))                                                           // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Empty\\)"
+		assert.Emptyf(t, len(b), "msg with args %d %s", 42, "42")                         // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Emptyf\\)"
+		assert.Empty(t, len(string(b)))                                                   // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Empty\\)"
+		assert.Emptyf(t, len(string(b)), "msg with args %d %s", 42, "42")                 // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Emptyf\\)"
+		assert.Empty(t, len([]string{"e"}))                                               // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Empty\\)"
+		assert.Emptyf(t, len([]string{"e"}), "msg with args %d %s", 42, "42")             // want "empty: remove unnecessary len%!\\(EXTRA string=assert, string=Emptyf\\)"
+		assert.Empty(t, string(str))                                                      // want "empty: remove unnecessary string conversion%!\\(EXTRA string=assert, string=Empty\\)"
+		assert.Emptyf(t, string(str), "msg with args %d %s", 42, "42")                    // want "empty: remove unnecessary string conversion%!\\(EXTRA string=assert, string=Emptyf\\)"
```

Comment:
When there is EXTRA there is a problem with the arguments passed to sprintf like function

Reply:
did not catch
provide example, please

Reply:
sorry, I do not understand, how this is relevant to this PR or this specific checker

the code like
```go
func myAssert(t *testing.T, args ...any) {
    t.Helper()
    assert.Equalf(t, "", str, args...)
}
```

will be transformed to
```go
func myAssert(t *testing.T, args ...any) {
    t.Helper()
    assert.Emptyf(t, str, args...)
}
```

do you want separate test case for it, or what?

also, the last changes in `formatter` cover some related cases

Reply:
I just checked, everything is fine now.

You fixed the issue that was present with EXTRA with this commit 89374dbd07216fcc848b19d7f3214bdd2322a6e0

Apparently here:
https://github.com/Antonboom/testifylint/pull/129/commits/89374dbd07216fcc848b19d7f3214bdd2322a6e0#diff-abd161b3072e56ba5d635c7937f319f4ad49e4d8f5857b6d88a4e096c8ea853aR113